shm/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Player.hpp"
+#include "Cargo.hpp"
+#include "Map.hpp"
+#include "Island.hpp"
+#include <iostream>
+#include <memory>
+#include <vector>
+#include <string>
+#include <utility>
+#include <time.h>
+
+// Return by value to give up ownership
+std::unique_ptr<Cargo> generateCargo() {
+    // https://portroyale3.fandom.com/wiki/Goods
```

Comment:
Love this game :) But the original first one form 200x :D

Reply:
Yeah, good old games.